src/extension.ts

```diff
@@ -25,12 +26,8 @@ export function activate(context: vscode.ExtensionContext) {
   console.log('Extension "vscode-peacock" is now active!');
 
   registerCommands();
-
-  State.recentColor = getCurrentColorBeforeAdjustments();
-
   addSubscriptions(context);
-
-  // context.subscriptions.push(disposable);
+  applyInitialConfiguration();
```

Comment:
should this be `await applyInitialConfiguration();` ?

Reply:
Hmm, it probably could be, but it doesn't have to be since nothing else is dependent on it. The activate method is currently not async. Can it be? I don't really know.